test/graphparser.cc

```diff
@@ -331,6 +334,16 @@ void Baltimore(const std::string& config_file) {
   sequence<OSMWay> ways(ways_file, false);
   ways.sort(way_predicate);
 
+  // check z-level
```

Comment:
did we get lucky and some of the tests just happened to have this tag in the data we were using?

Reply:
yes, it is turned out to be quite spread